cy/cypress/support/commands.js

```diff
@@ -132,6 +132,44 @@ Cypress.Commands.add("uninstall", (module) => {
     cy.execDrush(`-y pm:uninstall ${module}`)
 })
 
+Cypress.Commands.add("createTestUser", (testUser) => {
+    cy.execDrush(`ucrt ${testUser} --password=password`)
+    cy.execDrush(`user:role:add govcms_content_author ${testUser}`)
+})
+
+Cypress.Commands.add("removeTestFiles", () => {
+    // Remove all test files
+    cy.drupalLogin()
+    cy.visit('admin/content/media')
+    cy.get('.select-all > .form-checkbox').check()
+    cy.get('#edit-action').select('media_delete_action')
+    cy.get('#edit-submit').click()
+    cy.get('#edit-also-delete-file').check()
+    cy.confirm()
+
+})
+
+Cypress.Commands.add("deleteBlock", () => {
```

Comment:
This is a good encapsulation, but I believe it belongs in `blocks.cy.js` (probably after `cleanUp()`) as it is not used anywhere else. Putting it with the blocks will improve readability, and we can always move it later if need be.